### ISSUE TITLE
Tray Icon does not raise rssguard Layer to OnTop

### ISSUE THREAD

**Issue by AndyM48** [martinrotter/rssguard#1392]
### Brief description of the feature request

If rssguard is open but covered by other windows, clicking on the tray icon does not raise rssguard to `on top`. I think that may be it should.

Arch Linux: 6.8.7-artix1-2

**Comment by martinrotter** [comment]
If window is opened but not on top, then clicking tray icon should hide the window and another click should bring it open and on top.

**Comment by AndyM48** [comment]
Yes, of course, close the window and then reopen it has the same effect, but is a double click and not a single click.

If the window is closed, one click opens it, but if the window is hidden then, as it stands, two clicks are required. The state if the window, closed or hidden, is not apparent to the use, so the user does not know whether one or two clicks are required to reveal it.
